### PR TITLE
[WIP] Better http error handling

### PR DESCRIPTION
**Pull request by Kmaschta**
@Phocea Following https://github.com/marmelab/ng-admin/pull/1253

### CODE REVIEW

**Comment by Kmaschta** [comment]
@jpetitcolas There is still an issue with protractor or SauceLabs. Can you take a look?

**Comment by fzaninotto** [comment]
@Phocea the build fails, but it seems to be because of a problem in your code rather than a build artefact. Can you take a look?

**Comment by jpetitcolas** [comment]
Tests should be fixed now, **but** we still need to check new feature. I quickly copy/pasted README sample, but it doesn't boot the application anymore. Back to WIP, but we should not be too far. :)

File: src/javascripts/ng-admin/Main/component/provider/HttpErrorService.js [hunk 1]
```diff
@@ -1,38 +1,36 @@
...
-        displayError: text => notification.log(text, { addnCls: 'humane-flatty-error' }),
-    }
-}
+    displayError: text => notification.log(text, { addnCls: 'humane-flatty-error' }),
+});
 
 httpErrorService.$inject = ['$state', '$translate', 'notification'];
 
-export default httpErrorService;
+export default { $get: httpErrorService };
```
Comment (jpetitcolas) [inline src/javascripts/ng-admin/Main/component/provider/HttpErrorService.js]: @Phocea: this is a provider, hence we need a `$get` method. :)

**Comment by Phocea** [comment]
Replaced by https://github.com/marmelab/ng-admin/pull/1257

**Pull request closed without merging**